BridgeAppSDK/SBAUserWrapper.swift

```diff
@@ -85,6 +85,11 @@ public protocol SBAUserWrapper: class, SBBAuthManagerDelegateProtocol {
     var consentSignature: SBAConsentSignatureWrapper? { get set }
 
     /**
+     * Profile Image is stored in the keychain
+     */
+    var profileImage: UIImage? { get set }
```

Comment:
Please add this to the unit test mocks as well. Unit tests are broken by this change.